Workload-Security/Integration/aws-python-amazon-inspector-integration/README.md

```diff
@@ -0,0 +1,38 @@
+[![Launch Stack](https://cdn.rawgit.com/buildkite/cloudformation-launch-stack-button-svg/master/launch-stack.svg)](https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=c1-inspector-findings-protection&templateURL=https://vulnerabilitytestbucket.s3.amazonaws.com/c1-inspector-findings-protection.yaml)
```

Comment:
Please remove the other projects files from this PR

Reply:
Not sure why it picks up the other project file but I will redo it

Reply:
@wschadrach still there:

<img width="323" alt="image" src="https://user-images.githubusercontent.com/33869171/219793672-80adaa42-1321-43b2-b390-e7d10f16a82a.png">